src/KlaviyoAPI.php

```diff
@@ -441,6 +436,19 @@ protected function createRequestJson( $params )
         );
     }
 
+    /**
+     * Create query parameter array for CURL.
+     *
+     * @param array $params Key/value pairs to be set as query string.
+     * @return array
+     */
+    protected function createQueryParams($params)
```

Comment:
super nit: Should the spacing be updated from this PR to be uniform with this SDK?  
```suggestion
    protected function createQueryParams( $params )
```
and so forth for any changes in this PR?

Reply:
That's my fault for introducing that styling, it's wordpress convention and I was working on the WP plugin at the same time. There have been a few PRs made by other users of the SDK that have proposed adding a coding standard fixer which is definitely a good idea.